client/src/components/organisms/AdminRequestGroupBrowser.tsx

```diff
@@ -70,6 +73,21 @@ const AdminRequestGroupBrowser: FunctionComponent = () => {
         }
     }, [requestGroup]);
 
+    const deleteQuery = gql`
+        mutation deleteRequestGroup($id: ID) {
+            deleteRequestGroup(_id: $id) {
+                _id
+            }
+        }
+    `;
+    const [mutateDeleteRequestGroup] = useMutation(deleteQuery);
+
+    const deleteRequestGroup = async () => {
+        await mutateDeleteRequestGroup({ variables: { id: requestGroup?._id } });
+        // replace current page in browser so user cannot go back to non-existent requestGroup page
```

Comment:
🙌 this was a great clarificaiton comment I was literally about to search up what was happening with history